### PR TITLE
Updating readme and changelog

### PR DESCRIPTION
**Pull request by james-martin-jd**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+    - Dropdown
+    - Checkbox
+    - ColorPicker, using [react-color](https://casesandberg.github.io/react-color/)
+    - Header (an `<h2>` tag rendered inside of the form)
+    - Comment (a `<p>` tag rendered inside of the form)
+    - Decimal / Float / Integer fields
+    - Email field
+    - Phone field
+    - Multiple, a custom Form Model field that indicates the child element is repeatable
+
```
Comment (kevin-howard-jd) [inline CHANGELOG.md]: Include Group?
- Group. A ordered set of fields that can be treated as a single sub-form

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+### Updated
+
+- Notification class was updated to validate notifications when being read from an inbox. This way only compatible notifications are retrieved and the data is more predicatble for display purposes
+- Updated ACL class predicate from `acl:defaultForNew`, which is deprecated, to `acl:default`
+- Renamed the rdf predicate prefix from `a` to `rdf` to align with standard convention
+
+### Deprecated
+
+- The ShExFormBuilder component is now deprecated in favor of the new FormModel component. The component remains in the library for now, but the documentation has been removed
```
Comment (kevin-howard-jd) [inline CHANGELOG.md]: Space in "Form Model"?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,35 @@
...
+### Added
+
+- FormModel component
+  - New component that renders a form from a Form Model
+  - New components added to support the form model. The following list of components are all compatible with the Form Model:
+    - Textbox
+    - TextArea
+    - Date / DateTime / Time fields, using [react-datepicker](https://reactdatepicker.com/)
+    - Dropdown
+    - Checkbox
```
Comment (kevin-howard-jd) [inline CHANGELOG.md]: Explicitly say this is for a Boolean checkbox, not Tristate?

File: README.md [hunk 4]
```diff
@@ -162,25 +162,47 @@ Here's an example of a basic ProfileUploader component, demonstrating how to add
 
 We currently re-expose withWebId and LogoutButton so you can use the basic components without installing other libraries.
 
-### ShExFormBuilder
+### FormModel
 
-**Note**: This is an early preview build of a work-in-progress component and subject to large-scale changes in future releases.
-
-This component allows you to create a fully functional form, complete with submit, cancel, and validation rules, from a [ShEx](http://shex.io/shex-primer/index.html) Shape.
+This component takes a FormModel (build using the Form Language and [UI ontology](https://www.w3.org/ns/ui#)). A properly formatted FormModel will render a form, complete with constraints. Some things are not yet supported, such as ui:arc links, FormForms, and more. This component is still being improved and worked on.
```
Comment (kevin-howard-jd) [inline README.md]: It is not really that FormForm is not supposed, it is that ui:Choice and ui:Options are not supported.

File: README.md [hunk 5]
```diff
@@ -162,25 +162,47 @@ Here's an example of a basic ProfileUploader component, demonstrating how to add
...
 
 #### Supported Field Types:
 
 ##### Textbox
 
-Textboxes are the default input type for each field in a shape. In future versions, there will be a way to determine if you want a textbox vs a textarea based on properties such as maxlength.
+Textboxes are the default input type for each field in a shape, and represented in the FormModel by SingleLineTextFields. It can have common constraints like maxLength, as defined in the Form Language
```
Comment (kevin-howard-jd) [inline README.md]: Missing sentence period.

File: README.md [hunk 6]
```diff
@@ -162,25 +162,47 @@ Here's an example of a basic ProfileUploader component, demonstrating how to add
...
+
+##### Textarea
+
+A Textarea is represented in the FormModel by a MultiLineTextField, and similar to textbox has constraints defined by the Form Language
```
Comment (kevin-howard-jd) [inline README.md]: Missing sentence period.

File: README.md [hunk 7]
```diff
@@ -192,37 +214,52 @@ Validation is done using ShEx constraints. You can read more about these constra
...
-| successCallback   | Function | Function that writes a success message to console | Optional. Overrides the existing success callback and allows custom success functions.                                                      |
-| errorCallback     | Function | Function that writes the error message to console | Optional. Overrides the existing error callback function.                                                                                   |
-| messageValidation | Object   | null                                              | An Object containing an array of error strings. The error strings will be used in most non-validation situations.                           |
-| autoSaveMode      | Boolean  | false                                             | Optional. Determines of the form will autosave or have a save/cancel button. Default is save/cancel button.                                 |
+| Props         | Type     | Default                                           | Description                                                                                                                       |
```
Comment (kevin-howard-jd) [inline README.md]: Include sentence periods for each Description.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,35 @@
 # Solid React Components Library
 
+## 0.5.0 ( December 17, 2019 )
+
+### Added
+
+- FormModel component
```
Comment (kevin-howard-jd) [inline CHANGELOG.md]: Ending colon

File: README.md [hunk 9]
```diff
@@ -162,25 +162,47 @@ Here's an example of a basic ProfileUploader component, demonstrating how to add
...
+
+Numbers are represented in the form as a textbox with special custom constraints in place, enforcing that the textbox is an integer, float, or decimal depending on the Form Model definition.
+
+##### Checkbox
+
+A checkbox is rendered by a BooleanField in the Form Model, representing a binary yes/no choice. Tri-state checkboxes are not yet supported.
+
+##### Email / Phone field
+
+Email and Phone Number fields are custom textboxes, similar to the Number fields. They each take a regex pattern from the form model and
```
Comment (kevin-howard-jd) [inline README.md]: Missing end of sentence "... from the form model and" And what? I'm left in suspense...

Comment (james-martin-jd) [reply README.md]: Always keep them wanting more

File: CHANGELOG.md (hunk 2 above)
Comment (james-martin-jd) [reply CHANGELOG.md]: The name of the component is FormModel, as opposed to a Form Model. Maybe I can put the < > around it to make it clearer?

**Pull request merged**